addons/website_event_track/views/event_track_views.xml

```diff
@@ -100,10 +100,13 @@
             <search string="Event Tracks">
                 <field name="name"/>
                 <field name="tag_ids"/>
-                <field name="event_id"/>
+                <field name="partner_id" string="Speaker" filter_domain="[
```

Comment:
This is no-longer really "partner_id" then, let's make it a filter tag instead with "filter_speaker" as name.

Reply:
hey @awa-odoo, 

If we remove partner_id field and make it filter_speaker, how it will consider the user input for search as
we are inputting email, phone etc?

Please correct me if i understood your point wrong.

Reply:
Hello :wave: 

You're right, in search views we can only use the user input on "field" tags, I forgot about this limitation.
You can keep it as is then.

The upgrade runbot seems to be in error, would you mind rebasing? After that we should be good to go.

Thanks & cheers!